src/main/java/com/uwetrottmann/trakt5/TraktV2Interceptor.java

```diff
@@ -28,7 +28,7 @@ public Response intercept(Chain chain) throws IOException {
     public static Response handleIntercept(Chain chain, String apiKey,
             @Nullable String accessToken) throws IOException {
         Request request = chain.request();
-        if (!TraktV2.API_HOST.equals(request.url().host())) {
+        if (!(TraktV2.API_HOST.equals(request.url().host()) || TraktV2.API_STAGING_HOST.equals(request.url().host()))) {
```

Comment:
Can this use `trakt.apiHost()` as well?

Reply:
this is static context. If you change signature and make it instance method, then it will be possible to get rid of TraktV2.API_HOST and TraktV2.API_STAGING_HOST constants.

Reply:
I made 2 versions.